scripts/foundry/Utils.s.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.19;
+
+import "forge-std/Script.sol";
+import { Safe, Enum } from "safe/Safe.sol";
+import { MultiSend } from "safe/libraries/MultiSend.sol";
+import { ITreasury } from "borrow/interfaces/ITreasury.sol";
+
+/// @title Utils
+/// @author Angle Labs, Inc.
+contract Utils is Script {
+    /*//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+                                                       CONSTANTS                                                    
+    //////////////////////////////////////////////////////////////////////////////////////////////////////////////////*/
+
+    uint256 public constant BASE_18 = 1e18;
+
+    ITransmuter public transmuter = ITransmuter(address(0x0));
+    ITransmuter public treasuryArbitrum = ITreasury(0x0D710512E100C171139D2Cf5708f22C680eccF52);
```

Comment:
should we add an action in the TODO list for vaultManager deployment to add stuff in this repo?

Reply:
added